keywords.txt

```diff
@@ -726,6 +726,17 @@ PADC_TEMP	LITERAL1
 PADC_VREF	LITERAL1
 PADC_VBAT	LITERAL1
 
+ALT0	LITERAL1
+ALT1	LITERAL1
+ALT2	LITERAL1
+ALT3	LITERAL1
+ALT4	LITERAL1
+ALT5	LITERAL1
+ALT6	LITERAL1
+ALT7	LITERAL1
+PDUAL	LITERAL1
+PREMAP	LITERAL1
+
```

Comment:
Is it also required to add Pyx_ALTz  as literal ?

Reply:
I don't think